doc/man7/OSSL_PROVIDER-default.pod

```diff
@@ -255,6 +261,8 @@ The OpenSSL default provider supports these operations and algorithms:
 
 =item ED448, see L<EVP_KEYMGMT-ED448(7)>
 
+=for comment We don't yet have the corresponding MLKEM doc in the works.
```

Comment:
This should be a `TODO(ML-KEM)`

Reply:
Actually, turns out these are just additional names for a doc that is there, fixed.